worlds/stardew_valley/data/craftable_data.py

```diff
@@ -249,7 +254,7 @@ def create_recipe(name: str, ingredients: Dict[str, int], source: RecipeSource,
 charcoal_kiln = skill_recipe(Machine.charcoal_kiln, Skill.foraging, 2, {Material.wood: 20, MetalBar.copper: 2})
 
 crystalarium = skill_recipe(Machine.crystalarium, Skill.mining, 9, {Material.stone: 99, MetalBar.gold: 5, MetalBar.iridium: 2, ArtisanGood.battery_pack: 1})
-furnace = skill_recipe(Machine.furnace, Skill.mining, 1, {Ore.copper: 20, Material.stone: 25})
+furnace = skill_craftsanity_recipe(Machine.furnace, Skill.mining, 1, {Ore.copper: 20, Material.stone: 25})

```

Comment:
I feel like this is the kind of situation where it would be worth it to add a comment to explain why the furnace is not just a `skill_recipe` like the others. Otherwise, we won't remember when reworking the recipes to move them to content packs. 🫣